blocks/api/simple-dom.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import { forEach } from 'lodash';
+
+export function createSimpleNode( node, filter = Array ) {
+	if ( ! node ) {
+		return null;
+	}
+
+	if ( node.nodeType === 3 ) {
```

Comment:
This should either have a comment explaining the [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants), or referencing the node type constant `Node.TEXT_NODE` directly:

https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#Node_type_constants

I think previously these were avoided due to browser support but with increased browser support, we should be okay. I've confirmed the constants exist in IE11:

![IE11](https://user-images.githubusercontent.com/1779930/32277695-aa4a86fc-bee9-11e7-8ec5-82e01da41b1f.png)

Reply:
I know we've done that everywhere else, sorry. It's just a quick POC.